lib/mix/tasks/guardian_trackable.install.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Mix.Tasks.GuardianTrackable.Install do
+  @shortdoc "Generates a new migration for GuardianTrackable"
+
+  @moduledoc """
+  Generates a migration.
+
+  The repository must be set under `:ecto_repos` in the
+  current app configuration or given via the `-r` option.
+
+  This generator follows the functionality of `ecto.gen.migration`, so it will
+  be created in the configured migration direction with a timestamp-prefixed
+  filename.
+
+  ## Examples
+
+      mix guardian_trackable.install
+      mix guardian_trackable.install -r Custom.Repo
+      mix guardian_trackable.install --schema accounts
+
+  ## Command line options
+
+    * `-r`, `--repo` - the repo to generate migration for
+    * `--schema` - the schema to generate migration for
+  """
+  use Mix.Task
+
+  import Mix.Ecto
+  import Mix.Generator
+
+  @doc false
+  def run(args) do
+    {opts, _, _} = OptionParser.parse(args, switches: [schema: :string])
+
+    no_umbrella!("ecto.gen.migration")
+    repos = parse_repo(args)
+
+    Enum.each(repos, fn repo ->
```

Comment:
It seems unlikely that you'd want to generate the migration for more than one Repo, but since we have no way of detecting which repo is the *right* repo, I guess this is okay?

Reply:
Was just the way Ecto and GuardianDB did it (used those as reference), so figured there was a good/smart reason that I was unaware of for doing it that way.